index.html

```diff
@@ -880,6 +881,35 @@ <h3>The <dfn>ActionHandler</dfn> callback</h3>
       </p>
     </section>
 
+    <section> <h3>The <dfn>setActionHandler()</dfn> method</h3>
+      <p>
+        Takes <code>name</code> as string argument and <code>action</code> as argument of type <a>ActionHandler</a>. Sets the handler function for the specified <a>Action</a> matched by <code>name</code>. Throws on error. Returns a reference to the same object for supporting chaining.
+      </p>
+      <p>
+         The <code>action</code> callback function will implement an <a>Action</a> and SHOULD be called by implementations when a request for invoking the <a>Action</a> is received from the underlying platform.
+      </p>
+      <p>
+        There SHOULD be at most one handler for any given <a>Action</a> and newly added handlers replace the old handlers.
+      </p>
+      <p>
+        When an external request for invoking the <a>Action</a> identified by <var>name</var> is received, the runtime SHOULD execute the following steps:
```

Comment:
We probably should move this somewhere else in the next iteration.
A call can also come when only `addAction()` was used and the same applies.

Reply:
Right, I was thinking the same. It also applies to read, write and observe requests.